.github/workflows/test.yml

```diff
@@ -217,7 +217,6 @@ jobs:
       height_grep_text: 'finished initial sync to chain tip, using gossiped blocks sync_percent=100.* current_height=Height'
 
   lightwalletd-rpc-test:
-    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
```

Comment:
Question: why is this condition removed?

Reply:
It was wrongly place (and wrongly removed). Fixed now.